cromwellApiClient/src/main/scala/cromwell/api/model/TimeUtil.scala

```diff
@@ -0,0 +1,22 @@
+package cromwell.api.model
+
+import java.time.format.DateTimeFormatter
+import java.time.{OffsetDateTime, ZoneOffset}
+
+object TimeUtil {
+  /**
+    * Instead of "one of" the valid ISO-8601 formats, standardize on this one:
+    * https://github.com/openjdk/jdk/blob/jdk8-b120/jdk/src/share/classes/java/time/OffsetDateTime.java#L1886
+    */
+  private val Iso8601MillisecondsFormat = DateTimeFormatter.ofPattern("uuuu-MM-dd'T'HH:mm:ss.SSSXXXXX")
+
+  implicit class EnhancedOffsetDateTime(val offsetDateTime: OffsetDateTime) extends AnyVal {
+    /**
+      * Discards the original timezone and shifts the time to UTC, then returns the ISO-8601 formatted string with
+      * exactly three digits of milliseconds.
+      */
+    def toUtcMilliString: String = Option(offsetDateTime).map(
```

Comment:
Why do we need two copy/pastes of this code?

Reply:
`cromwell-api-client` despite being in the multi-project does not use project dependencies, including [`common`](https://github.com/broadinstitute/cromwell/commit/6eb104b44a03eed972c886e28a95cefa4632d019#diff-c01844a8d491e55a2a182f351df7f99c) where the other `TimeUtil` was added in this PR.

So like [`WorkflowStatus`](https://github.com/broadinstitute/cromwell/blob/40/cromwellApiClient/src/main/scala/cromwell/api/model/WorkflowStatus.scala#L5-L10), CAC needs its own copy of the util.

Reply:
🤦‍♂